.test-infra/jenkins/job_PostCommit_Java_ValidatesRunner_Dataflow_Java11.groovy

```diff
@@ -20,26 +20,40 @@ import CommonJobProperties as commonJobProperties
 import PostcommitJobBuilder
 
 
-PostcommitJobBuilder.postCommitJob('beam_PostCommit_Java11_ValidatesRunner_Dataflow',
+PostcommitJobBuilder.postCommitJob('beam_PostCommit_Java_ValidatesRunner_Dataflow_Java11',
   'Run Dataflow ValidatesRunner Java 11', 'Google Cloud Dataflow Runner ValidatesRunner Tests On Java 11', this) {
 
   description('Runs the ValidatesRunner suite on the Dataflow runner with Java 11 worker harness.')
 
-  commonJobProperties.setTopLevelMainJobProperties(delegate, 'master', 270)
+  def JAVA_11_HOME = '/usr/lib/jvm/java-11-openjdk-amd64'
+  def JAVA_8_HOME = '/usr/lib/jvm/java-8-openjdk-amd64'
 
+  commonJobProperties.setTopLevelMainJobProperties(delegate, 'master', 270)
   publishers {
     archiveJunit('**/build/test-results/**/*.xml')
   }
 
   steps {
+    gradle {
+      rootBuildScriptDir(commonJobProperties.checkoutDir)
+      tasks(':runners:google-cloud-dataflow-java:testJar')
+      tasks(':runners:google-cloud-dataflow-java:worker:legacy-worker:shadowJar')
+      switches("-Dorg.gradle.java.home=${JAVA_8_HOME}")
+    }
+    
     gradle {
       rootBuildScriptDir(commonJobProperties.checkoutDir)
       tasks(':runners:google-cloud-dataflow-java:validatesJava11Runner')
```

Comment:
Why do we have a [validatesJava11Runner](https://github.com/apache/beam/blob/a39b69cc9ed5c930e59c863ccb66ba61351fe1e3/runners/google-cloud-dataflow-java/build.gradle#L322) task? (Shouldn't we just invoke the [validatesRunner](https://github.com/apache/beam/blob/a39b69cc9ed5c930e59c863ccb66ba61351fe1e3/runners/google-cloud-dataflow-java/build.gradle#L316) task?)

Reply:
Running a Dataflow validatesRunner with Java 11 requires setting an additional task parameter in order to have correct worker harness image (`java.specification.version`). Since I don't think we can parametrize Gradle tasks, we're stuck with two of them

Reply:
Shouldn't java.specification.version come automatically from the JVM and if we are using a Java 11 virtual machine, it should already provide java.specification.version of 11 while a Java 8 virtual machine would specify 8?

Reply:
You're right, the whole duplicate is redundant. I'll fix this